packages/plugin-conversation/src/interceptor.js

```diff
@@ -24,27 +29,22 @@ export default class ConversationInterceptor extends Interceptor {
    * @returns {Promise<Object>}
    */
   onRequest(options) {
-    if (!options.method || options.method.toUpperCase() === `GET`) {
-      return options;
-    }
-
-    return this.spark.device.isSpecificService(`conversation`, options.service || options.uri)
-      .then((isConversationService) => {
-        if (!isConversationService) {
+    return this.shouldEncryptRequest(options)
+      .then((shouldDecrypt) => {
+        if (!shouldDecrypt) {
           return options;
```

Comment:
for consistency, i've changed them all to Promise.resolve, but in in this case, it doesn't matter since the last line of each method controls the resolution value